uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -177,7 +176,7 @@ private void assertResponse(OrchestratorZoneResponse expectedResponse, Orchestra
             assertNotNull(actualConnectionDetails);
             assertEquals(expectedConnectionDetails.getSubdomain(), actualConnectionDetails.getSubdomain());
             assertEquals(expectedConnectionDetails.getUri(), actualConnectionDetails.getUri());
-            assertEquals(expectedConnectionDetails.getDashboardUri(), actualConnectionDetails.getDashboardUri());
+            assertNotNull(actualConnectionDetails.getDashboardUri());
```

Comment:
Same comment as for MockMvc tests. At least assert that dashboard url in actualConnectionDetails contains a base dashboard url + "#/login"

Reply:
Done